packages/common/tests/aws.js

```diff
@@ -107,7 +107,7 @@ test('pullStepFunctionEvent returns original message if message not on S3', asyn
       execution_name: 'execution'
     },
     meta: {
-      bucket: 'test bucket',
```

Comment:
I'm unclear why this change was needed for eslint to pass, I didn't modify this file at all.